### PR TITLE
add task

### PR DESCRIPTION
**Pull request by vmisai**
[DEMO LINK](https://vmisai.github.io/layout_search-bar-airbnb/)
[TEST REPORT LINK](https://vmisai.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,10 +7,29 @@
...
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
+    <form data-qa="small" class="search_form form_small">
+      <label class="search_label search_label_small">
+        <input
+
+          class="search_input search_input_small"
```
Comment (DarkMistyRoom) [inline src/index.html:28]:
```suggestion
        <input
          class="search_input search_input_small"
```

File: src/index.html [hunk 2]
```diff
@@ -7,10 +7,29 @@
...
+      <label class="search_label search_label_big">
+        <input
+          data-qa="keypress"
+          class="search_input search_input_big"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
+    </form>
+    <form data-qa="small" class="search_form form_small">
```
Comment (DarkMistyRoom) [inline src/index.html:24]:
```suggestion
    </form>
    
    <form data-qa="small" class="search_form form_small">
```

Comment (DarkMistyRoom) [reply src/index.html:24]: Add empty lines between multiline sibling blocks of HTML

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,142 @@
+/*Basic styling*/
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```
Comment (DarkMistyRoom) [inline src/style.css:7]: Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser to read your
web document

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+.form_big {
+  width: 100%;
+  height: 70px;
+}
+
+.form_small {
+  width: 100%;
+  height: 42px;
+}
```
Comment (DarkMistyRoom) [inline src/style.css:50]: don't duplicate the `width`. Move it to the common class

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+
+.search_label_small:after {
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
```
Comment (DarkMistyRoom) [inline src/style.css:88]: same here. Move duplicated styles to the common class

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
Good job! But avoid duplications

**Review by anastasiia-tilikina — CHANGES_REQUESTED** [review]
check my comments

File: src/index.html [hunk 6]
```diff
@@ -7,10 +7,29 @@
...
+    </form>
+
+    <form data-qa="small" class="search_form form_small">
+      <label class="search_label search_label_small">
+        <input
+          class="search_input search_input_small"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+      </label>
```
Comment (VitaliyBodnar) [inline src/index.html:32]: let's try to use BEM naming 
```suggestion
    <form data-qa="small" class="search search--small">
      <label class="search__label search__label--small">
        <input
          class="search__input search__input--small"
          type="text"
          placeholder="Try &quot;Los Angeles&quot;"
        >
      </label>
```
https://en.bem.info/methodology/quick-start/

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,124 @@
...
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+/*Fonts*/
```
Comment (VitaliyBodnar) [inline src/style.css:12]: remove all comments

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,124 @@
...
+
+.search_input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: inherit;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
```
Comment (VitaliyBodnar) [inline src/style.css:82]: 61, 78, 97 is duplicated, so please move all duplicated styles to the variables

**Review by VitaliyBodnar — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+  font-weight: 300;
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 600;
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
```
Comment (loralevitska) [inline src/style.css:21]: Remove redundant default styles
```suggestion
  font-weight: 300;
  src: url("./fonts/Avenir-Book.ttf") format("truetype");
}

@font-face {
  font-family: "Avenir";
  font-weight: 600;
  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
```

**Review by loralevitska — APPROVED** [review]
Well done)

**Pull request closed without merging**